.env-example

```diff
@@ -18,9 +18,9 @@ export SYSTEM_GITHUB_TOKEN=""
 # FormSG keys
 export SITE_CREATE_FORM_KEY=""
 
-# Required to connect to DynamoDB
-export AWS_ACCESS_KEY_ID=""
-export AWS_SECRET_ACCESS_KEY=""
+# Required to connect to DynamoDB and Amplify
+export AWS_ACCESS_KEY_ID="" # Generate your own access key and secret access key from AWS
```

Comment:
nit: I think just commenting it once is enough on top of the lines

Reply:
da7aad3